classes/task/process_duplicated_event_visibility_change.php

```diff
@@ -113,7 +113,7 @@ public function execute() {
             $event = $apibridge->get_already_existing_event([$data->duplicatedeventid]);
             if (!$event || !in_array($event->status, ['EVENTS.EVENTS.STATUS.PROCESSED', 'EVENTS.EVENTS.STATUS.PROCESSING_FAILURE'])
                 || count($event->publication_status) == 0
-                || count($event->publication_status) == 1 && $event->publication_status[0] === 'internal') {
+                || (count($event->publication_status) == 1 && $event->publication_status[0] === 'internal')) {
```

Comment:
Just to be sure, this is only for readability, and you didn't encounter a bug concerning this, right? Because the behaviour shouldn't be changed by this